spec/models/report_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Report do
+  let!(:user) { create(:user) }
+
+  describe '#editable?' do
+    let!(:report) { create(:report, user: user) }
+
+    context 'when the user is the owner of the report' do
+      it 'is editable' do
+        expect(report.editable?(user)).to eq true
+      end
+    end
+
+    context 'when the user is not the owner of the report' do
+      let!(:other_user) { create(:user) }
+
+      it 'is not editable' do
+        expect(report.editable?(other_user)).to eq false
+      end
+    end
+  end
+
+  describe '#created_on' do
+    it 'returns the date when the report was created' do
+      travel_to Time.zone.local(2023, 8, 30, 01, 04, 44) do
```

Comment:
```suggestion
      travel_to "2023-08-30 01:04:44".in_time_zone do
```

みたいに書いた方が人間は読みやすいかも？

Reply:
そっちの方がいいですね！